README.md

```diff
@@ -13,11 +13,36 @@ Parameters in some Income Tax (Making Tax Digital) APIs map to box numbers in [S
 **Note:** The date shown is the date that the change was released to Sandbox or Production.
 
 ---
+
+### 15 May 2024 
+
+The following change is now available in Sandbox.
+
+#### Individuals Other Income API 
+
+Existing API version 1.0 updated with the following changes.
+
+##### Added
+
+Create and Amend Other Income endpoint:
+
+Added validation for `postCessationReceipts` parameters in `TY 2023-24 or later` schema:
+- `amount`:
+    - values cannot be negative
+- `taxYearIncomeToBeTaxed`:
+    - format must be YYYY-YY
+    - range must be exactly one year
+    - minimum year 2010, maximum year 2099 
```

Comment:
I don't think we need to spell out or repeat the actual validation here (single source of truth).

```suggestion
Added missing validation for the following `postCessationReceipts` fields in the `TY 2023-24 or later` schema as per the API documentation:
- `amount`
- `taxYearIncomeToBeTaxed`
```